linkerd/app/outbound/src/lib.rs

```diff
@@ -93,7 +93,7 @@ impl<S> Outbound<S> {
         self.stack.into_inner()
     }
 
-    fn no_tls_reason(&self) -> tls::NoClientTls {
+    pub fn no_tls_reason(&self) -> tls::NoClientTls {
```

Comment:
Once we remove `no_tls_reason` from the gateway, this can be private again
```suggestion
    fn no_tls_reason(&self) -> tls::NoClientTls {
```